cli/src/pcluster/cli/commands/cluster_logs.py

```diff
@@ -65,18 +70,40 @@ def execute(self, args: Namespace, extra_args: List[str]) -> None:  # noqa: D102
         try:
             if args.output_file:
                 self._validate_output_file_path(args.output_file)
+
+            is_pcluster_bucket = not args.bucket
+
+            if is_pcluster_bucket:
+                # Validate the bucket prefix for both the default pcluster bucket
+                # and CustomS3Bucket if specified in the cluster configuration.
+                # Skip validation if a bucket is specified via the --bucket CLI argument.
+                self._validate_bucket_prefix(args.bucket_prefix)
+
             return self._export_cluster_logs(args, args.output_file)
         except Exception as e:
             utils.error(f"Unable to export cluster's logs.\n{e}")
             return None
 
+    @staticmethod
+    def _validate_bucket_prefix(bucket_prefix: str) -> None:
```

Comment:
This function is equal to the one used for the image builder part. Can we remove code duplication?

Reply:
Done